core/lib/dal/migrations/20250122102800_contract-verifier-new-schema.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS contract_verification_info_v2 (
+    initial_contract_addr BYTEA NOT NULL PRIMARY KEY,
+    bytecode_keccak256 BYTEA NOT NULL,
+    bytecode_without_metadata_keccak256 BYTEA,
+    verification_info JSONB NOT NULL,
+
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Add hash indexes for hash columns
+CREATE INDEX IF NOT EXISTS contract_verification_info_v2_bytecode_keccak256_idx ON contract_verification_info_v2 USING HASH (bytecode_keccak256);
+CREATE INDEX IF NOT EXISTS contract_verification_info_v2_bytecode_without_metadata_keccak256_idx ON contract_verification_info_v2 USING HASH (bytecode_without_metadata_keccak256);
```

Comment:
nit: In practice hash index speeds `select` up a bit but slows down `inserts` significantly so we generally use btree in our codebase. Up to you as for this table insert performance doesn't matter much

Reply:
Huh, that's a really nice catch, I didn't know that. Will change